### ISSUE TITLE
supervisors not starting correctly on testarteria1

### ISSUE THREAD

**Issue by Smeds** [arteria-project/arteria-provisioning#45]
sudo supervisorctl status all
bcl2fastq-ws:bcl2fastq-ws-10900   FATAL     Exited too quickly (process log may have details)
runfolder-ws:runfolder-10800      FATAL     Exited too quickly (process log may have details)
siswrap-ws:siswrap-ws-10700       FATAL     Exited too quickly (process log may have details)

**Comment by johandahlberg** [comment]
This is probably due to switching usernames on part of the Ansible deployment, but not all of it...

**Comment by withrocks** [comment]
Do we have reproduction steps? Did this happen directly after `vagrant up` or had some changes occurred?

**Comment by withrocks** [comment]
Strike that, reproduced directly by `vagrant up`.

**Comment by brainstorm** [comment]
The issue seems to be about permissions:

```
[root@testarteria1 supervisor]# tail bcl2fastq-ws-10900-stderr---supervisor-9o9wow.log
  File "/usr/local/lib/python2.7/logging/config.py", line 794, in dictConfig
    dictConfigClass(config).configure()
  File "/usr/local/lib/python2.7/logging/config.py", line 576, in configure
    '%r: %s' % (name, e))
ValueError: Unable to configure handler 'file_handler': [Errno 13] Permission denied: '/var/log/arteria/bcl2fastq-ws.log'

[root@testarteria1 supervisor]# ls -alh /var/log/arteria/
totalt 184K
drwxr-xr-x   2 arteria root 4,0K 22 okt 12.42 .
drwxr-xr-x. 11 root    root 4,0K  2 nov 13.40 ..
-rw-r--r--   1 root    root  28K 28 okt 14.47 bcl2fastq-ws.log
-rw-r--r--   1 root    root 138K 22 okt 14.17 runfolder.log
-rw-r--r--   1 root    root    0 22 okt 12.42 siswrap.log
```

But the interesting bit is that you seem to have it well defined in your ansible roles on `arteria-provisioning/ansible-st2/roles/arteria_core/tasks/main.yml`:

```
- name: ensure /var/log/arteria dir exists
  file:
    state: directory
    path: /var/log/arteria
    owner: "{{ arteria_user }}"
```

Where `arteria user is`:

```
arteria_user: arteria
arteria_uid: 4242
arteria_group: arteria
arteria_gid: 4242
```

So I guess that another process changes it back to `root` at some point? `chown arteria:arteria /var/log/arteria` solves the issue, but I'm still intrigued on why ansible does not enforce those permissions in that directory across `vagrant provision`'s.

**Comment by withrocks** [comment]
I thought https://github.com/arteria-project/arteria-provisioning/pull/47 had fixed this (although I forgot to close the ticket). @brainstorm, do you have latest?

**Comment by withrocks** [comment]
Pretty sure this was closed by #47. Just reopen if this is still an issue.
